src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToUseAnalyzers.cs

```diff
@@ -24,42 +24,51 @@ public GivenThatWeWantToUseAnalyzers(ITestOutputHelper log) : base(log)
         }
 
         [Theory]
-        [InlineData("WebApp", false)]
-        [InlineData("WebApp", true)]
-        [InlineData("WebApp", null)]
-        public void It_resolves_requestdelegategenerator_correctly(string testAssetName, bool? isEnabled)
+        [MemberData(nameof(OffByDefaultGeneratorTestData))]
+        public void It_resolves_offbydefaultgenerator_correctly(GeneratorSpec generator, bool? isEnabled)
         {
             var asset = _testAssetsManager
-                .CopyTestAsset(testAssetName, identifier: isEnabled.ToString())
+                .CopyTestAsset("WebApp", identifier: isEnabled.ToString())
                 .WithSource()
                 .WithProjectChanges(project =>
                 {
                     if (isEnabled != null)
                     {
                         var ns = project.Root.Name.Namespace;
-                        project.Root.Add(new XElement(ns + "PropertyGroup", new XElement("EnableRequestDelegateGenerator", isEnabled)));
+                        // Delete when we have a new targeting pack that contains Microsoft.Extensions.Configuration.Binder.SourceGeneration.
+                        if (generator == GeneratorSpec.ConfigurationBinding)
+                        {
+                            project.Root.Add(new XElement(ns + "ItemGroup", new XElement("Analyzer", new XAttribute("Include", "does_not_yet_exist\Microsoft.Extensions.Configuration.Binder.SourceGeneration.dll"))));
```

Comment:
```suggestion
                            project.Root.Add(new XElement(ns + "ItemGroup", new XElement("Analyzer", new XAttribute("Include", "does_not_yet_exist\\Microsoft.Extensions.Configuration.Binder.SourceGeneration.dll"))));
```
Make sure you build and run tests locally before trusting my PR suggestions 😄

Reply:
Righto 😅 I was having issues with building and seeing test failures even from the tip of main, so I wanted to try my luck in CI.

Reply:
You know... it's Friday 😅